hindsight.yaml

```diff
@@ -10,15 +10,21 @@ Parameters:
     MinLength: 8
     MaxLength: 128
     Default: 5bcfd3fb5cba35da5f1be3c347bc
-  DataBucketName:
+  DataBucketPrefix:
     Description: Bucket name where data is persisted
```

Comment:
```suggestion
    Description: Bucket name that will be postfixed with the environment.
```

Reply:
How you're using it changed a bit, so I figure we should update the description.

Reply:
done